rhizome/lib/vm_setup.rb

```diff
@@ -199,7 +227,7 @@ def cloudinit(unix_user, public_key, private_subnets, ip4)
     match:
       macaddress: #{yq(guest_mac)}
     dhcp6: true
-    #{"dhcp4: true" if ip4}
+    #{"dhcp4: true" if private_ipv4}
```

Comment:
isn't `private_ipv4` tantamount to having ipv4 at all? Do you like having the extra bytes in `private_ipv4` symbol around to make it easier to search for?

Reply:
Nope, with this version, we can just have private ipv4. This way, even if the customers don't have public ipv4, they can start using private ipv4 to communicate in between their vms.